### PR TITLE
Use asciidoc as the manual page source format

### PR DESCRIPTION
**Pull request by dkopecek**
To generate the manual page, you can use the a2x utility
from the asciidoc package:

 $ a2x -v -f manpage clevis.1.adoc
 ...

or use the provided doc/Makefile:

 $ cd doc/
 $ make

### CODE REVIEW

**Comment by dkopecek** [comment]
I've added a separate commit which implements the automatic manual page generation.

File: doc/clevis-decrypt.1.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+CLEVIS-DECRYPT(1)
+=================
+:doctype: manpage
+
+== NAME
+
+clevis-decrypt - Decrypts using the policy defined at encryption time
```
Comment (npmccallum) [inline doc/clevis-decrypt.1.adoc]: Please use spaces instead of dashes.

Comment (dkopecek) [reply doc/clevis-decrypt.1.adoc]: I think it's a convention to use dashes in the NAME section as it refers to the manual page name. See for example `man git apply`. The SYNOPSIS section on the other hand refers to the usage of the actual command.

Comment (npmccallum) [reply doc/clevis-decrypt.1.adoc]: Thanks!

**Pull request merged**